source/command-monitoring/tests/unified/command.yml

```diff
@@ -0,0 +1,94 @@
+description: "commands"
+
+schemaVersion: "1.0"
+
+createEntities:
+  - client:
+      id: &client client
+      observeEvents:
+        - commandStartedEvent
+        - commandSucceededEvent
+        - commandFailedEvent
+  - database:
+      id: &database database
+      client: *client
+      databaseName: &databaseName command-monitoring-tests
+  - collection:
+      id: &collection collection
+      database: *database
+      collectionName: &collectionName test
+
+initialData:
+  - collectionName: *collectionName
+    databaseName: *databaseName
+    documents:
+      - { _id: 1, x: 11 }
+
+tests:
+  - description: "A successful command"
+    operations:
+      - name: distinct
```

Comment:
I switched to used `distinct` instead of `count` here because:
1) drivers don't have `count` API methods anymore
2) it seemed best to avoid any dependency on the recent changes around `estimatedDocumentCount` implementations